stumpy/gpu_stump.py

```diff
@@ -79,7 +87,7 @@ def _compute_and_update_PI_kernel(
     σ_Q : numpy.ndarray
         Standard deviation of the query sequence, `Q`
 
-    k : int
+    profile_len : int
```

Comment:
Can we use `l` in place of `profile_len` here?

Reply:
I will replace `profile_len` with `w` in `stumpy/gpu_stump.py`.